frontend/app/src/admin/Admin.tsx

```diff
@@ -0,0 +1,8 @@
+import { observer } from 'mobx-react-lite';
+import React from 'react';
+
+export const AdminPage = observer(() => (
+  <>
+    <div>Admin</div>
+  </>
```

Comment:
I dont think we need the empty div <></> if we already have a <div></div> wrapping Admin

Reply:
> This should also be included in its own PR not sure why it is in this branch

I didn't get it

Reply:
these are two unrelated changes

Page added if user goes to deleted bounty from URL
and 
Added Empty Admin Page